src/api/cvc4cpp.cpp

```diff
@@ -2353,6 +2353,23 @@ Term Solver::mkBVFromStrHelper(uint32_t size,
   return mkValHelper<CVC4::BitVector>(CVC4::BitVector(size, val));
 }
 
+Term Solver::mkCharFromStrHelper(std::string s) const
+{
+  CVC4_API_CHECK(s.find_first_not_of("0123456789abcdefABCDEF", 0)
+                     == std::string::npos
+                 && s.size() <= 5 && s.size() > 0)
+      << "Unexpected string for hexidecimal character " << s;
+  std::stringstream hexString;
+  hexString << s;
+  uint32_t val;
+  hexString >> std::hex >> val;
```

Comment:
The following would be a bit simpler:

```suggestion
  uint32_t val = static_cast<uint32_t>(std::stoul(s, 0, 16));
```